providers/os/connection/snapshot/blockdevices.go

```diff
@@ -151,31 +151,55 @@ func (blockEntries BlockDevices) FindDevice(name string) (BlockDevice, error) {
 }
 
 // Searches all the partitions in the device and finds one that can be mounted. It must be unmounted, non-boot partition
-// If multiple partitions meet this criteria, the largest one is returned.
-func (device BlockDevice) GetMountablePartition() (*PartitionInfo, error) {
+func (device BlockDevice) GetMountablePartitions(includeBoot bool) ([]*PartitionInfo, error) {
```

Comment:
nit: 
```suggestion
func (device BlockDevice) GetMountablePartitions(includeAll bool) ([]*PartitionInfo, error) {
```

you're not just including the boot, but any arbitrary partitions that we find